### DISCUSSION TITLE
Need to know what is mean of the acronym WPSS

### DISCUSSION THREAD

**Discussion by BobbyBoucher001** [community/community#61363] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

There is nothing on what WPSS stands for. Any help would be greatly apreaciated.

**Comment by Hsin-Hung** [comment]
Where did you see the acronym?

**Reply by BobbyBoucher001** [reply]
Thank you for your response, I just found the french version and was able to translate it.
you can Find it here: https://github.com/wet-boew/wet-boew-wpss/wiki, There is a French page, and it has the acronym in full,
https://github.com/wet-boew/wet-boew-wpss/wiki/Outil-de-validation-des-Services-de-planification-et-des-normes-Web
Once I read the French version the translation was easy peasy =)

Outil de validation des Services de planification et des normes Web (SPNW)

Web Planning Services and Standards (WPSS) 
I hope this helps,
Again thank you for your response.
**Cheers**
TheWaterBoy :)
